app/mbm/views.py

```diff
@@ -146,6 +158,29 @@ def get_route(self, source_vertex_id, target_vertex_id):
             ]
         }
 
+    def format_distance(self, dist_in_meters):
+        """
+        Given a distance in meters, return a tuple (distance, time)
+        where `distance` is a string representing a distance in miles and
+        `time` is a string representing an estimated travelime in minutes.
+        """
+        meters_per_mi = 1609.344
+        dist_in_mi = dist_in_meters / meters_per_mi
+        formatted_dist = round(dist_in_mi, 1)
+        # Don't worry about single-mile case since we always report at least
+        # one decimal (i.e. "1.0 miles")
+        dist_unit_str = 'miles'
+        distance = f'{formatted_dist} {dist_unit_str}'
+
+        # Assume 8mph as a naive guess of speed
+        mi_per_min = 8 / 60
```

Comment:
In the issue you mentioned a blog post that estimated 11-18. Anecdotally, that's closer to what I've observed from my own rides. Google seems to be using something around 10mph for a few routes I've looked at around Chicago (although I have no idea if that's constant or what factors they look at).

Reply:
That said, I think the most important thing is also showing the distance to the user, which you're doing!

Reply:
The few blog posts I've read (e.g. [How accurate are Google maps cycling time estimates?](http://www.betterbybicycle.com/2014/09/how-accurate-are-google-maps-cycling.html)) seem to have a consensus that 10mph is the baseline and then they adjust that somewhat mysteriously based on factors like elevation and number of stops. I feel like we have pretty good grounds to bump this up to 10.